### ISSUE TITLE
Simple Condition

### ISSUE THREAD

**Issue by JonasTietz** [BehaviorTree/Groot#31]
Hi,
this is not really an issue, but I think it would be a nice feature, which would get rid of some redundant work. I would like to be able to create an condition node in Groot were I could just specify a boolean in the blackboard to check if it is true or false. Right now I always write a very simple function and use the registerSimpleCondition call with the port list.

**Comment by facontidavide** [comment]
Hi,

it make sense. I will work on that

**Comment by hlzl** [comment]
@JonasTietz 
Have you tried using the `BlackboardCheckString` decorator? It should be able to do a similar behavior. You could specify the port as _value_B_, `TRUE` or `FALSE` as _value_A_ and `FAILURE` as desired NodeStatus with _return_on_mismatch_.

**Comment by JonasTietz** [comment]
No I did not consider that. 
Correct me if I am wrong, but I would still need an node, which would always return success and is not a decorator, because a decorator always needs a child.
I think that it would be a bit hacky. But otherwise it seems like an valid workaround for now.

**Comment by hlzl** [comment]
> Correct me if I am wrong, but I would still need an node, which would always return success and is not a decorator, because a decorator always needs a child.
> I think that it would be a bit hacky. But otherwise it seems like an valid workaround for now.

I agree, it is a bit hacky, especially when you need an extra child. I only use it when I have an action and want to check a related condition beforehand. In this case, you can put it right in front of the action node.
